pyface/ui/qt4/layout_widget.py

```diff
@@ -0,0 +1,133 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+from enum import Enum
+
+from traits.api import provides
+
+from pyface.qt import QtGui
+from pyface.i_layout_item import DEFAULT_SIZE, SizePolicy
+from pyface.i_layout_widget import ILayoutWidget, MLayoutWidget
+from pyface.ui.qt4.widget import Widget
+
+
+#: Maximum widget size (some versions of PyQt don't export it)
```

Comment:
PyQt or PyQt5? We don't need to add new code with support for PyQt as we've dropped support for it.

Reply:
It's some but not all versions of PyQt5 - I don't remember which.